VPC-Section-Formatting (#141)
C2Devel/terraform-provider-croccloud: docs/resources/route.md

# Resource: aws_route

> Creates a routing table entry (a route) in the VPC routing table.

Reviewer: вот не уверен насчет артикля, в VPC  можно создать несколько таблиц маршрутизации

}
```

## Argument reference

The following arguments are supported:

* `route_table_id` - (Required, Forces new resource, String) The ID of the routing table.

> One of the following destination arguments must be supplied:

Reviewer: только один есть вообще

The following arguments are supported:

* `route_table_id` - (Required, Forces new resource) (String) The ID of the routing table.

One of the following destination arguments must be supplied:

* `destination_cidr_block` - (Required, Forces new resource) (String) The destination CIDR block.

One of the following target arguments must be supplied:

> * `gateway_id` - (Optional) (String) The ID of an internet gateway.

Reviewer: ```suggestion
* `gateway_id` - (Optional) (String) The ID of the internet gateway.
```

The following arguments are supported:

* `route_table_id` - (Required, Forces new resource, String) The ID of the routing table.

One of the following destination arguments must be supplied:

* `destination_cidr_block` - (Required, Forces new resource, String) The destination CIDR block.

One of the following target arguments must be supplied:

* `gateway_id` - (Optional, String) The ID of the internet gateway.
* `network_interface_id` - (Optional, String) The ID of the network interface.
* `transit_gateway_id` - (Optional, String) The ID of the transit gateway.

> These arguments are **deprecated** and should not be used:

Reviewer: ```suggestion
These argument is **deprecated** and should not be used:
```

~> **Note** Only the arguments that are configured (one of the above) will be exported as an attribute once the resource is created.

> * `id` - (String) Route identifier computed from the routing table identifier and route destination.

Reviewer: ```suggestion
* `id` - (String) The route identifier computed from the routing table identifier and route destination.
```

* `instance_owner_id` - (String) The project ID that owns the EC2 instance.

> * `origin` - (String) How the route was created - `CreateRouteTable`, `CreateRoute` or `EnableVgwRoutePropagation`.

Reviewer: ```suggestion
* `origin` - (String) Describes how the route was created - by `CreateRouteTable`, `CreateRoute` or `EnableVgwRoutePropagation`.
```
